OlinkAnalyze/vignettes/LOD.Rmd

```diff
@@ -0,0 +1,129 @@
+  fig.width = 6,
+  fig.height = 3,
+  message = FALSE,
+  warning = FALSE,
+  time_it = TRUE,
+  fig.align = "center"
+)
+
+library(OlinkAnalyze)
+library(dplyr)
+
+```
+
+## Introduction
+
+This vignette describes how to use Olink® Analyze to integrate Limit of Detection (LOD) into Explore HT and Explore 3072 datasets. Although it is recommended to use all Explore data in downstream analyses, LOD information can be useful when performing technical evaluations of a dataset.
+
+In this vignette, you will learn how to use `olink_lod()` to add LOD information to your Explore dataset.
+
+
+## Integrating LOD
+
+Limit of Detection (LOD) is a metric that indicates the lowest measurable value of a protein. LOD can be helpful when performing technical evaluations of NPX datasets, such as calculating CVs. 
+
+LOD can be added to Explore NPX datasets using `olink_lod()`. This function can incorporate LOD into an Explore dataset based on either an NPX dataset’s negative controls or using predetermined fixed LOD values, which Olink Support can provide upon request. The default LOD calculation method for Explore NPX data is based off of a dataset's negative controls. 
+
+In both normalization methods, an LOD and PCNormalizedLOD column are integrated into the dataset. If an NPX file is PC normalized, the LOD and PCNormalizedLOD columns will contain identical values. If an NPX file is intensity normalized, the intensity normalized LOD will be reported in the LOD column and the PC normalized LOD will be reported in the PCNormalizedLOD column. An Explore dataset's NPX or PCNormalizedNPX values should be compared against the LODs that are integrated using the `olink_lod()` function.  
+
+## Setup Explore datasets
+
+Explore datasets are standard Olink® Explore HT and Explore 3072 NPX tables. They can be loaded using the `read_NPX()` function with default Olink Software Explore NPX parquet files as input.
+
+
+```{r dataset_generation, eval = FALSE, message=FALSE, warning=FALSE}
+explore_npx <- read_NPX("~/Explore_NPX_file.parquet")
+```
+
+
+## Integrating Negative Control LOD
+
+The negative control (NC) LOD method requires at least 10 negative controls in a dataset. Negative control data is available in the standard exported Explore HT and Explore 3072 NPX parquet files. 
+
+A negative control will not contribute to the count of the minimum required NCs if either of the following apply:
+
+ + The negative control contains an assay QC warning across all assays, excluding Olink's internal control assays
+ + The negative control does not pass sample QC criteria (sample QC failure or warning) in all Explore HT blocks or all Explore 3072 panels
+
+For each assay, the median NPX is calculated for all negative controls present in a project. For Explore 3072, overlapping assays are assessed separately, within their respective panels. The LOD for each assay is then calculated as the assay's median negative control NPX value plus 0.2 Standard Deviations. 
```

Comment:
@kathy-nevola 
median negative control NPX value plus 0.2 Standard Deviations.  or 

3 Standard Deviations or 0.2 whichever higher 
?

Reply:
Good catch, fixed this